fluentui_core/src/main/java/com/microsoft/fluentui/view/MSRecyclerView.kt

```diff
@@ -26,7 +26,7 @@ import android.view.ViewConfiguration
  * Note: The crazy math part has been taken from Android framework Scroller.java and modified
  * to match our needs.
  */
-internal open class MSRecyclerView : RecyclerView {
+open class MSRecyclerView : RecyclerView {
```

Comment:
I am assuming we didn't want to expose this outside library.
If this is done only for cross module use , mark it's scope as library only

Reply:
Done. Will be available in next patch